api/src/db/models/staging/certificates.py

```diff
@@ -1,6 +1,15 @@
+import uuid
+
+from sqlalchemy import Uuid, text
+from sqlalchemy.orm import Mapped, mapped_column
+
 from src.db.models.legacy_mixin.certificates_mixin import TcertificatesMixin
 from src.db.models.staging.staging_base import StagingBase, StagingParamMixin
 
 
 class Tcertificates(StagingBase, TcertificatesMixin, StagingParamMixin):
     __tablename__ = "tcertificates"
+
+    tcertificates_id: Mapped[uuid.UUID | None] = mapped_column(
+        Uuid(as_uuid=True), server_default=text("gen_random_uuid()")
+    )
```

Comment:
It might not matter, but for consistency could we use the UUID type from `from sqlalchemy.dialects.postgresql`- that's what we've done elsewhere / should be the default anyways. It probably is just an alias, but to be safe/ consistent.

Also, we should make the column unique to be safe + adds an index for us to query against.

```suggestion
    tcertificates_id: Mapped[uuid.UUID | None] = mapped_column(
        UUID, server_default=text("gen_random_uuid()"), unique=True
    )
```